sandbox/pages/StatusWizardStepperPage.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Components 0.1
+
+Item {
+    id: root
+    width: 800
+    height: 100
+
+    //Simulate animation between steps
+    property bool stepsCompleted: false
+    function reset() {
+        for (var i = 0; i < stepsListModel.count; i++) {
+            stepsListModel.setProperty(i, "loading", false);
+            stepsListModel.setProperty(i, "stepCompleted", false);
+        }
+    }
+
+    ListModel {
+        id: stepsListModel
+        ListElement {description:"Send Request"; loadingTime: 0; loading: false; stepCompleted: false}
+        ListElement {description:"Receive Response"; loadingTime: 2000; loading: false; stepCompleted: false}
+        ListElement {description:"Confirm Identity"; loadingTime: 2000; loading: false; stepCompleted: false}
+    }
```

Comment:
How does this behave when the first and third item are `stepCompleted: true` ?

Shouldn't the component rather track at which step it is? E.g. being at step 2 implies that step 1 is completed. Or are we also dealing with cases where one can be at a certain step without having completed a prior one?